wwwroot/config.json

```diff
@@ -26,16 +26,16 @@
          
         // Pop-up disclaimer that will be shown at the start
         // The text of the disclaimer is in lib/Views/GlobalDisclaimer.html
-        // globalDisclaimer: {
-        //     confirmationRequired: false,
-        //     buttonTitle: "I agree",
-        //     title: "Disclaimer",
-        //     // If the current browser location is not prod, or is dev, then show a "not the real site" warning.
-        //     // The text of that is in lib/Views/DevelopmentDisclaimerPreamble.html
-        //     //prodHostRegex: "gov.\\.au$",
-        //     //devHostRegex: "\\b(staging|preview|test|dev)\\.",
-        //     enableOnLocalhost: true // If false, don't show this when running on localhost
-        // },
+        globalDisclaimer: {
```

Comment:
Nice - shall we leave global disclaimer commented out, but keep the changes in index.js as they're guarded by the config parameter check? What do you think? I can make this change if you're happy w/ it

Reply:
Merging with this in mind, but revert/let me know if you think otherwise

Reply:
Oops just saw this, yup I think leaving the part in `index.js` uncommented should be fine